core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -156,7 +156,8 @@ oppia.directive('audioBar', [
 
           $scope.doesCurrentAudioTranslationNeedUpdate = function() {
             if (!isAutogeneratedLanguageCodeSelected()) {
-              return getAudioTranslationInCurrentLanguage().needsUpdate;
+              var audioTranslation = getAudioTranslationInCurrentLanguage();
+              return (audioTranslation && audioTranslation.needsUpdate);
```

Comment:
Should there be an additional check for undefined here? Since, one of the errors mentioned was that ausioTranslation could be undefined.